.env.example

```diff
@@ -1,7 +0,0 @@
-DB_USERNAME=postgres
```

Comment:
Why did you delete this file?

Reply:
I felt it's not needed since i already gitignore the `.env file` already which my feature is using

Reply:
We still need it. It's what shows anybody that will work on the project the `.env` needed to get the app running.

Reply:
Oh, Alright. I would do likewise